src/main/java/games/strategy/engine/framework/startup/ui/PlayerComboBoxSelector.java

```diff
@@ -0,0 +1,129 @@
+package games.strategy.engine.framework.startup.ui;
+
+import java.awt.Container;
+import java.awt.GridBagConstraints;
+import java.awt.Insets;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.swing.JCheckBox;
+import javax.swing.JComboBox;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+
+import games.strategy.engine.data.PlayerID;
+import games.strategy.engine.data.properties.GameProperties;
+import games.strategy.triplea.Constants;
+
+class PlayerComboBoxSelector {
```

Comment:
:+1: for extracting and cleaning up this class.

I'm not sure having `ComboBox` in the name even made sense before this change because this class has evolved to encapsulate much more than just a combo box.  Would `PlayerSelector` be any better?